auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -320,16 +343,28 @@ def get_crop_transforms(self):
                 max_samples_per_class = None
             indices_key = None
 
+            sigmoid = self.extra_options.get("sigmoid", False)
+            crop_add_background = self.crop_params.get("crop_add_background", False)
+
+            if crop_ratios is None:
+                crop_classes = output_classes
+                if sigmoid and crop_add_background and self.class_index is not None and len(self.class_index)>1:
```

Comment:
What's this `self.class_index` used for? Only for the check?

Reply:
yes, to check